assets/javascripts/initializers/add-canned-replies-ui-builder.js.es6

```diff
@@ -34,11 +34,18 @@ export default {
   initialize(container) {
     const siteSettings = container.lookup("site-settings:main");
     const currentUser = container.lookup("current-user:main");
-
+    let currentUserGroupNames;
```

Comment:
```suggestion
    let currentUserGroupNames = [];
```

`currentUserGroupNames` can stay `undefined` here if `!currentUser.groups` (which is very unlikely to happen), but just to be on the safe side (might be a problem on line 46).